src/components/Offers/Edit/EditOfferSchema.js

```diff
@@ -59,8 +59,6 @@ export default yup.object().shape({
         .required(HumanValidationReasons.REQUIRED),
     location: yup.string()
         .nullable(true)
-        .matches(/^([a-zA-Z]+([\s-][a-zA-Z])*)+, ([a-zA-Z]+([\s-][a-zA-Z])*)+$/,
```

Comment:
Does the new api not follow the format? Why should we remove validation?

Reply:
By having the validation here the request never reached the location service, I just removed it so we could have the request. I cannot think of any useful validation other than to check for strings. Any suggestions?

Reply:
Hmm that's weird, before we had the validation and still sent the field to the API, right?

Reply:
Yes, but we would send the field in the format `city, country` to the old API: that format is not recognized by the new Location API. I can make it so it is recognized but personally i do not believe it should be the case: the API should receive any piece of textual data and return the closest match it finds, without needing a specific format.

Reply:
I just remembered, whatever is applied to the edit schema should be applied to the creation schema

Reply:
Fair enough, it makes sense if the API supports all formats